resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -633,13 +633,13 @@ spec:
         params:
           - filePattern: "**/whitesource-ip.json"
             type: whitesource-ip
-          - filePattern: "**/whitesource-riskReport.pdf"
+          - filePattern: "whitesource/*risk-report.pdf"
```

Comment:
As for me it looks correct, but for example, in detectExecuteStep filePatten looks like this `**/*BlackDuck_RiskReport.pdf`. Maybe it is better to have the same way of matching files across different steps. But this way also works fine